build/document-extractor.js

```diff
@@ -575,6 +575,11 @@ function _addSectionProse($) {
       }
     }
   }
+
+  if (id) {
+    id = id.replace(/_+$/g, "");
+  }
```

Comment:
To be honest, this is not really where we don't to do this.

However, I believe those ids are generated as part of our m2h conversion.

It appears that GitHub's own parser doesn't generate perfect ids in this case either: https://github.com/mdn/content/blob/main/files/en-us/web/css/value_definition_syntax/index.md#curly-braces-- 🤔

Reply:
I guess we could configure a custom plugin in the m2h processor to do this, but KISS tells me to just go with this solution.